CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# Changelog
+
```

Comment:
@AgustinBettati @oarbusi can you confirm adding this header won't affect the changelog update process?

Reply:
I think [this](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/7dabad93bcb6d2d72500bf29429a1db1d35804e0/scripts/update-changelog-unreleased-section.sh#L54) will be affected and should be modified. 
My understanding is that the command extracts the first line of the changelog file if it contains the pattern "## ", and now the first line is different. Please confirm @AgustinBettati because you have more context on this script

Reply:
Correct, this change would be overwritten once the unreleased section is updated. Format was defined as per the [changelog specification](https://developer.hashicorp.com/terraform/plugin/best-practices/versioning#version-headers) and how other providers define it so I believe we can leave it as is for now.

Reply:
discussed offline with @AgustinBettati .  I'll revert the changes in doc file.   This wouldn't have any effect as it'd be removed next time file is updated.

as a nice to have we can do it in a following PR at some moment

Reply:
thank you both, reverting change to this file